setup.rst

```diff
@@ -32,6 +32,9 @@ executable that needs to be installed on your system only once:
 
     # Windows systems
     c:\> php -r "readfile('https://symfony.com/installer');" > symfony
+    
+    # Windows 10 and xampp (whit the "PATH" environment variable defined in "c:\xampp\php'
+    c:\> php.exe -r "readfile('https://symfony.com/installer');" > symfony
```

Comment:
not needed, when `c:\xampp\php` is in your PATH, `php` will be resolved to `c:\xampp\php\php.exe` by Windows